packages/hooks/src/useEscrowZap.ts

```diff
@@ -218,6 +222,10 @@ interface UseEscrowZapProps {
   safetyValveDate: Date;
   details?: `0x${string}` | null;
   enabled?: boolean;
-  networkConfig?: { [key: number]: NetworkConfig }; // to override the default network config
+  networkConfig?: {
```

Comment:
I'm a little confused why two types of NetworkConfig is being used in this file. Can DM just use NETWORK_CONFIG from SmartInvoice directly?

Reply:
They have different different network_config values for some of the properties